--- ./app/wp-content/mu-plugins/000-debug/debug-mode.php	2019-12-18 13:56:13.000000000 -0500
+++ ./app/wp-content/mu-plugins/000-debug/debug-mode.php	2019-12-18 13:56:13.000000000 -0500
@@ -39,8 +39,9 @@
 	$is_nocache = isset( $_COOKIE['vip-go-cb'] ) && '1' === $_COOKIE['vip-go-cb'];
 	$is_debug = isset( $_COOKIE['a8c-debug'] ) && '1' === $_COOKIE['a8c-debug'];
 	$is_proxied = \is_proxied_request();
+	$is_local = ( ( defined( 'VIP_GO_ENV' ) && 'local' === VIP_GO_ENV ) ? true : false );
 
-	if ( $is_nocache && $is_debug && $is_proxied ) {
+	if ( ( $is_nocache && $is_debug && $is_proxied ) || $is_local ) {
 		return true;
 	}
 
--- ./app/wp-content/mu-plugins/debug-bar.php	2019-12-18 13:56:13.000000000 -0500
+++ ./app/wp-content/mu-plugins/debug-bar.php	2019-12-18 13:56:13.000000000 -0500
@@ -15,6 +15,10 @@
 		return true;
 	}
 
+	if ( defined( 'VIP_GO_ENV' ) && 'local' === VIP_GO_ENV ) {
+		return true;
+	}
+
 	return $enable;
 }, PHP_INT_MAX );
 
